### ISSUE TITLE
Protein vector readability

### ISSUE THREAD

**Issue by serbulent** [hkmztrk/SMILESVecProteinRepresentation#4]
Created protein vectors saved in a list by the getligprotvec.py in order to protein names in smiles_sample.txt. For big files it creates a readability problem. A dictionary might help for this issue such as ['protname1':[vectorvalue1, vectorvalue2], ['protname2':[vectorvalue1, vectorvalue2] ... ]

**Comment by hkmztrk** [comment]
Thanks for the feedback and suggestion!

Could you tell about what kind of readability problem occurs? Does it take too long to read or something else?

**Comment by serbulent** [comment]
No, the issue is not technical, what I mean is you should map them one by one to your original protein list to access them later by using accesion id and it might be error prone for user. If it is saved in a dictionary then you can access them more easily.

**Comment by hkmztrk** [comment]
It assumes you are using the same order given in your input, but yes saving in a dictionary might make things easier. 

I'll look into it when I find time. Thanks.

**Comment by serbulent** [comment]
You are welcome :)
